cms/tests/test_templatetags.py

```diff
@@ -128,7 +128,7 @@ class FakeRequest:
         template = '{% load cms_tags %}{% page_attribute page_title %}'
         output = self.render_template_obj(template, {}, request)
         self.assertNotEqual(script, output)
-        self.assertEqual(escape(script), output)
+        self.assertEqual(strip_tags(script), output)
```

Comment:
I feel like we need a second check to be more explicit about the content here. Something to prevent regression and verifies the content renders as we want.

Reply:
Isn't `page_attribute page_title` *exclusively* intended to be used for the `<title>…</title>`-tag? Then according to the [specs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title#technical_summary) we should sanitize it so that the permitted content "is not inter-element [whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace)".

Then using `escape(…)` is an absolute wrong approach. `strip_tags` is better but still not according to the specs. How could we sanitize the whitespace?

Reply:
We could rely on Django's auto-escape: If we do not escape or strip tags, Django will escape the result. This should be backwards compatible (we should check) and allow the developer to use the appropriate filter like `strip_tags` or `escape` or `strip_tags`.

Reply:
I think I have to retract my comment 🙁: It seems that while auto-escape works, still `{% page_attribute page_title as title %}{{ title|striptags }}` does not work. It seems auto-escape is also executed when assigning the tag's result to the `title` context var. In this example, `title` already contains the escaped title.

So, I would suggest to **not** change this and use `{{ request.current_page.get_page_title|striptags }}` as a work-around.